README.rst

```diff
@@ -0,0 +1,98 @@
+Introduction
+============
+
+.. image:: https://readthedocs.org/projects/adafruit-circuitpython-slideshow/badge/?version=latest
+    :target: https://circuitpython.readthedocs.io/projects/slideshow/en/latest/
+    :alt: Documentation Status
+
+.. image:: https://img.shields.io/discord/327254708534116352.svg
+    :target: https://discord.gg/nBQh6qu
+    :alt: Discord
+
+.. image:: https://travis-ci.org/adafruit/adafruit_CircuitPython_Slideshow.svg?branch=master
+    :target: https://travis-ci.org/adafruit/adafruit_CircuitPython_Slideshow
+    :alt: Build Status
+
+CircuitPython helper library for displaying a slideshow of images on a board with a built-in
+display.
+
+Dependencies
+=============
+This driver depends on:
+
+* `Adafruit CircuitPython <https://github.com/adafruit/circuitpython>`_
+
+Please ensure all dependencies are available on the CircuitPython filesystem.
+This is easily achieved by downloading
+`the Adafruit library and driver bundle <https://github.com/adafruit/Adafruit_CircuitPython_Bundle>`_.
+
+Usage Example
+=============
+
+.. code-block:: python
+
+    import adafruit_slideshow
+
+    slideshow = adafruit_slideshow.SlideShow()
+    slideshow.loop = False
+    slideshow.order = slideshow.ALPHA
+
+    while slideshow.update():
+        pass
```

Comment:
`ALPHA` is in the `PlayBackMode` class now. Check the rest of the example as well.